releasenotes/notes/sourcecode-setuptools-b869c154bbb897d9.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds source code integration with setuptools build metadata.
```

can we add just a bit more about why a user should care about this? Something like

```suggestion
    Adds source code integration with setuptools build metadata. This enables traces to be automatically tagged with version information to track deployments in Datadog... and more...
```

---

oh actually this isn't the case yet, we're just setting ourselves up to do this. Instead we should say that we are adding `import ddtrace.sourcecode.setuptools_auto` which will inject source code metadata into the package (which we will integrate with to tag data after)